packages/test/src/browser/view/test-view-contribution.ts

```diff
@@ -274,6 +274,14 @@ export class TestViewContribution extends AbstractViewContribution<TestTreeWidge
             commandId: TestViewCommands.SELECT_DEFAULT_PROFILES.id,
             order: 'aaaaa'
         });
+
+        menus.registerSubmenu([...PLUGIN_TEST_VIEW_TITLE_MENU, TestViewCommands.RUN_ALL_TESTS.id], '', undefined, undefined, undefined, {
+            'testing.profile.context.group': 'run'
+        });
+
+        menus.registerSubmenu([...PLUGIN_TEST_VIEW_TITLE_MENU, TestViewCommands.DEBUG_ALL_TESTS.id], '', undefined, undefined, undefined, {
+            'testing.profile.context.group': 'debug'
+        });
```

Comment:
Having `undefined, undefined, undefined` here is a good point to support my argument in favor of objects :)